src/docs/Components/RatingStars/code.mdx

```diff
@@ -159,12 +203,35 @@ These modifiers are to be applied **to the main container** :
 
 <Preview path="stars-result-all-sizes" />
 
+#### With a label
+
+You have the possibility to display a text explicitly telling the user the result of the note.
```

Comment:
You have the possibility to display a text explicitly telling the user the result of the note. =>
`You have the possibility (you are able) to display the average score or the total review count.

Reply:
Fixed